webapp/channels/src/packages/mattermost-redux/src/reducers/entities/limits.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {combineReducers} from 'redux';
+
+import {LimitsTypes} from 'mattermost-redux/action_types';
+import type {GenericAction} from 'mattermost-redux/types/actions';
+
+function usersLimits(state = {}, action: GenericAction) {
+    switch (action.type) {
+    case LimitsTypes.RECIEVED_USERS_LIMITS: {
```

Comment:
typo, it's spelled "received" and not "recieved"

Reply:
Oh no, it's back!

We used to have this typo in literally every single action, and it was consistently misspelled until I got sick enough to fix it in https://github.com/mattermost/mattermost/pull/2101 😛